modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,28 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
-a|* `<image>` - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+|`aggregatedClusterRoles`    |The aggregation of several cluster roles for a cluster-scoped instance. | `__<empty>__` |
+
+|`applicationSet` |The ApplicationSet Controller configuration options. | `_<object>_`
+a|* `<image>` - The container image for the ApplicationSet Controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
```

Comment:
```suggestion
a|* `<image>` - The container image for the ApplicationSet controller. This property overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
```
Per IBM Style, avoid ambiguous pronoun references: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references

Reply:
Discussed it further with Laura on Slack so I am not fixing this comment to maintain consistency for the word "Controller" throughout the GitOps documentation.